src/System.IO.Compression.Brotli/README.md

```diff
@@ -0,0 +1,214 @@
+    public BrotliStream(Stream baseStream, CompressionMode mode, bool leaveOpen, int bufferSize, CompressionLevel quality);
+    public BrotliStream(Stream baseStream, CompressionMode mode, bool leaveOpen, int bufferSize, CompressionLevel quality, uint windowSize);
+    public override bool CanRead { get; }
+    public override bool CanSeek { get; }
+    public override bool CanTimeout { get; }
+    public override bool CanWrite { get; }
+    public override long Length { get; }
+    public override long Position { get; set; }
+    public override int ReadTimeout { get; set; }
+    public override int WriteTimeout { get; set; }
+    protected override void Dispose(bool disposing);
+    public override void Flush();
+    protected virtual void FlushEncoder(bool finished);
+    public override int Read(byte[] buffer, int offset, int count);
+    public override long Seek(long offset, SeekOrigin origin);
+    public override void SetLength(long value);
+    public override void Write(byte[] buffer, int offset, int count);
+}
+```
+
+```Write``` write and compress bytes to stream, can only be called in Compress mode
+
+```Read``` read bytes from stream and write them to buffer, can only be called in Decompress mode
+
+Default ```CompressionLevel``` is ```Optimal```(max compression ratio).
+
+### Examples
+#### Compress
+```C#
+public void CompressData(byte[] data, MemoryStream compressed)
+{
+    using (var compressor = new BrotliStream(compressed, CompressionMode.Compress, true))
+    {
+        compressor.Write(data, i, chunkSize);
+        compressor.Dispose();
+    }
+}
+```
+
+After calling method compressed data will be in ```MemoryStream compressed``` .
+
+#### Decompress
+Decompress and write data to file.
+```C#
+private void DecompressDataToFile(byte[] compressedData, string outFile)
+{
+    MemoryStream compressed = new MemoryStream(compressedData);
+    FileStream fs = File.Create(outFile);
+    using (BrotliStream decompressBrotli = new BrotliStream(compressed, CompressionMode.Decompress))
+    {
+        decompressBrotli.CopyTo(fs);
+    }
+    fs.Dispose();
+}
+```
+
+#### Dynamic Web Compression
+You can add code like this to your Global.asax.cs in ASP.NET application to support dynamic compress
+```C#
+public static bool IsBrotliSupported()
+{
+    string AcceptEncoding = HttpContext.Current.Request.Headers["Accept-Encoding"];
+    if (!string.IsNullOrEmpty(AcceptEncoding) &&
+            (AcceptEncoding.Contains("brotli") || AcceptEncoding.Contains("br")))
+        return true;
+    return false;
+}
+
+public static void BrotliEncodePage()
+{
+    HttpResponse Response = HttpContext.Current.Response;
+
+    if (IsBrotliSupported())
+    {
+        string AcceptEncoding = HttpContext.Current.Request.Headers["Accept-Encoding"];
+        Response.Filter = new System.IO.Compression.BrotliStream(Response.Filter,        System.IO.Compression.CompressionMode.Compress);
```

Comment:
Please remove the extra spacing between Response.Filter and System.IO.Compression.CompressionMode.Compress.